src/armor_layers.cpp

```diff
@@ -305,29 +308,65 @@ void player::sort_armor()
             } else {
                 selected = leftListIndex;
             }
+        } else if (action == "EQUIP_ARMOR") {
+            // filter inventory for all items that are armor/clothing
+            int pos = g->inv_for_unequipped(_("Put on:"), [](const item &it) {
+                        return it.is_armor();
+                    });
+            // only equip if something valid selected!
+            if(pos != INT_MIN) {
+                // wear the item, if error, show it
+                if(!wear(pos)) {
+                    debugmsg("Failed trying to equip item @index [%d]", pos);
```

Comment:
This is actually *very* likely, try wearing something while already wearing power armor. Or try wearing a second pair of shoes etc.